Xamarin.Forms.Platform.iOS/Extensions/Extensions.cs

```diff
@@ -10,13 +10,13 @@ public static void ApplyKeyboard(this IUITextInput textInput, Keyboard keyboard)
 			textInput.AutocapitalizationType = UITextAutocapitalizationType.None;
 			textInput.AutocorrectionType = UITextAutocorrectionType.No;
 			textInput.SpellCheckingType = UITextSpellCheckingType.No;
+			textInput.KeyboardType = UIKeyboardType.Default;
```

Comment:
This affect `Chat` and `Text` as well as `CustomKeyboard`. I'd assume they all expect to modify a `UIKeyboardType.Default` keyboard. So this fixes the chat and text cases as well. Happy day!

Reply:
Yeah, that's right